README.md

```diff
@@ -46,14 +46,18 @@ export IBMCLOUD_API_KEY=<ibm-cloud-api-key>
 
 ## Usage
 
-```shell
+```
 $ ./bin/collect -h
-Usage of C:\MyStuff\Governance\np-guard\cloud-resource-collector\bin\collect:
+Usage of ./bin/collect:
   -out string
         file path to store results
   -provider string
         cloud provider from which to collect resources
+  -region value
+        cloud region from which to collect resources
 ```
+* Value of `-provider` must be either `ibm` or `aws`

```

Comment:
The help should print the list of regions we have defined, or at least offer an option to get the list (something like `--get-regions`). The reason is that the names of regions can differ and the user may not have the list of region names easily accessible.

Reply:
Added a flag.